lib/src/base-styles/_nx-lists.scss

```diff
@@ -103,13 +104,13 @@
 
     .nx-list__text {
       display: inline-block;
-      margin-left: -6px;
+      margin-left: -3px;
```

Comment:
This should probably have a comment explaining why it's not to spec and that browsers are inconsistent about the bullet spacing.

Reply:
> This should probably have a comment explaining why it's not to spec and that browsers are inconsistent about the bullet spacing.

Sure: https://github.com/sonatype/sonatype-react-shared-components/pull/157/commits/c4276f520a49e4a58ffc96fb24256663d00fdc67